docs/support/faq.md

```diff
@@ -136,97 +124,81 @@ Yes! See [boolean logic](../sdk/access-control/condition-types/boolean-logic) fo
-
-Each node only holds a private key share. When a user wants to decrypt something, he presents the thing to decrypt, and proof that he meets the conditions (a wallet signature). Each node independently checks that the user meets the condition with a RPC call to an ETH node. If the user meets the condition, the node uses its private key share to create a decryption share. The user collects the decryption shares and accumulates them above the threshold, and is then able to decrypt the content.
+Each node only holds a private key share. When a user wants to decrypt or sign something, they present the thing to decrypt and proof that they meet the conditions (using a wallet signature or permitted auth method). Each node independently checks that the user meets the condition with an RPC call to the applicable network. If the user meets the condition, the node uses its private key share to create a decryption os signing share. The user collects the decryption shares and accumulates them above the threshold and is then able to decrypt or sign the content.
 
 So, you can see, the nodes don't talk to each other when decrypting the content. Each node's private key share never leaves the node.
 
-### 4. How do new nodes that come online discover the key shares they need to help decrypt previously-encrypted data? Or is each key fragment tied to a single node lifetime?
-
-**(cont'd) If the latter, then doesn't that mean that there's no redundancy on the key fragments and you're very susceptible to nodes going offline? Like if too many nodes go offline then you might no longer have enough key fragments in the online nodes to decrypt some pieces of content?**
+### 4. How do new nodes that come online discover the key shares they need to help decrypt previously-encrypted data?
 
-Right now, we're running all the nodes, so the nodes and shares don't change. soon, we will run a federated network with named nodes, and the ultimate goal is to run a permissionless one. we use a process called proactive secret sharing to share the private key shares with new nodes as they come online. the shares given to new nodes are incompatible with any nodes that have left the network. we use threshold encryption with a 2/3 threshold so redundancy is built in.
+Right now, we're running the nodes for the Manzano and Habanero networks, which are decentralized networks being run by 3rd party operators. The nodes and shares for these networks don't change. Soon, we will run a federated network with named nodes, and the ultimate goal is to run a permissionless one. We use a process called proactive secret sharing to share the private key shares with new nodes as they come online. The shares given to new nodes are incompatible with any nodes that have left the network. We use threshold encryption with a 2/3 threshold, so redundancy is built-in.
```

Comment:
this sentence is a little misleading / confusing. You start by saying we are running the nodes and then say they're being run by 3rd parties and then say we'll soon be running a federated network...

i think we can instead say that manzano and habanero are federated networks being run by named 3rd party operators and that the ultimate goal is to be fully permisionless. Root keys are updated every epoch which makes old shares functionally useless, and new shares are shared to new operators as they join through proactive secret sharing. Ending with the sentence on threshold encryption is solid!

Reply:
Ah agree @a1ttech 

Updated to this -
Right now, Manzano and Habanero are federated networks being run by named 3rd party operators. The ultimate goal is to transition to a fully permissionless network. Root keys are updated every epoch, rendering old shares functionally useless. New shares are shared with new operators through proactive secret sharing. The network uses threshold encryption with a 2/3 threshold, providing redundancy and security.